docs/docs/intro.md

```diff
@@ -8,9 +8,9 @@ Discover **expressly** _in less than 5 minutes_.
 
 ---
 
-First, head over to [developer.fastly.com](https://developer.fastly.com) to get started with JavaScript on Fastly's Compute@Edge:
+First, head over to [developer.fastly.com](https://developer.fastly.com) to get started with JavaScript on Fastly Compute:
 
-1. Learn about Compute@Edge: [developer.fastly.com/learning/compute/](https://developer.fastly.com/learning/compute/)
+1. Learn about Compute: [developer.fastly.com/learning/compute/](https://developer.fastly.com/learning/compute/)
```

Comment:
I feel like this could use the "Fastly" prefix...

```suggestion
1. Learn about Fastly Compute: [developer.fastly.com/learning/compute/](https://developer.fastly.com/learning/compute/)
```